src/isar/mission_planner/echo_planner.py

```diff
@@ -118,57 +147,12 @@ def _create_drive_task(self, tag_name: str) -> DriveToPose:
         drive_task: DriveToPose = DriveToPose(pose=robot_pose)
         return drive_task
 
-    def _create_measurement_tasks(
-        self, tag_name: str
-    ) -> List[Union[TakeImage, TakeThermalImage]]:
-        """
-        Retrieve measurement type corresponding to inspection of a given tag. For now, this is a temporary hard-coded
-        solution.
-        """
-        try:
-            predefined_measurement_type: List[str] = predefined_measurement_types[
-                tag_name
-            ]
-        except KeyError:
-            self.logger.warning(
-                f"Tag not in predefined_measurement_types, will use default. Tag: {tag_name}"
-            )
-            predefined_measurement_type = ["Image"]
-
-        measurement_tasks: List[Union[TakeImage, TakeThermalImage]] = []
-        for measurement_type in predefined_measurement_type:
-            if measurement_type == "ThermalImage":
-                measurement_tasks.append(
-                    self._create_thermal_image_task(tag_name=tag_name)
-                )
-
-            elif measurement_type == "Image":
-                measurement_tasks.append(self._create_image_task(tag_name=tag_name))
-
-            else:
-                self.logger.exception(
-                    f"Invalid measurement type in predefined_measurement_types. Tag: {tag_name}, measurement type: {measurement_type}"
-                )
-
-        measurement_tasks = [task for task in measurement_tasks]
-
-        return measurement_tasks
-
-    def _create_image_task(self, tag_name: str) -> TakeImage:
-        tag_position_robot: Position = self._get_tag_position_robot(tag_name=tag_name)
-
-        image_task: TakeImage = TakeImage(
-            target=tag_position_robot,
-            tag_id=tag_name,
-        )
-        return image_task
-
-    def _create_thermal_image_task(self, tag_name: str) -> TakeThermalImage:
-        tag_position_robot: Position = self._get_tag_position_robot(tag_name=tag_name)
-
-        thermal_image_task: TakeThermalImage = TakeThermalImage(
-            target=tag_position_robot,
-            tag_id=tag_name,
-        )
 
-        return thermal_image_task
+def echo_sensor_type_to_isar_inspection_task(
```

Comment:
We could make this function a member function of the class, and prefix it with '_' since it is only used internally
```suggestion
@staticmethod
def _echo_sensor_type_to_isar_inspection_task(
```